interface-definitions/container.xml.in

```diff
@@ -519,6 +519,12 @@
               <multi/>
             </properties>
           </leafNode>
+          <leafNode name="disable-dns">
```

Comment:
The common term in VyOS we use for DNS on the cli would be `name-server`.
Please change the CLI node to `no-name-server`.

Please also update the smoketests we use for containers at https://github.com/vyos/vyos-1x/blob/current/smoketest/scripts/cli/test_container.py and include a new `test_` that defines this option and reads back the generated config file and probes that the key is indeed `false`.

Other then that the change looks legit.

Reply:
I've made the changes suggested but haven't pushed them yet.  The reason being is I can't seem to run the smoke test's.  I posted in the forum the [error I'm seeing](https://forum.vyos.io/t/how-to-properly-build-and-smoketest-vyos/12198/3) (with no response since I posted on Friday).  Can I get access the the slack channel then maybe I can work through the smoke tests problems in real time?